### ISSUE TITLE
aria-invalid test case is insufficient

### ISSUE THREAD

**Issue by JamesCatt** [accessibilitysupported/a11ysupport.io#138]
This test case only tests on `<input type="text">` and lists SRs as supporting it based only on this test. But in my own testing I've discovered that both Talkback and iOS Voiceover support it on this type of input but not others.

Talkback appears to support only `type="text"` and `textarea` elements, but not `radio`, `checkbox` or `select`. iOS doesn't support `select`. I haven't yet tested others such as `password` or `date`.

With this in mind, listing both these screen readers as "supporting" is misleading.

**Comment by mfairchild365** [comment]
Hi @JamesCatt - Thank you for raising this issue!

It takes a long time to run a full test across all of the AT/browser combos, so I try to keep the test cases to a minimum (but hopefully representative sample). It would be impossible to manually test every way that `aria-invalid` can be used. That's why only `<input type="text">` was tested.

I wasn't aware of the poor support for certain other form controls. I agree that the current support data is misleading.

The first step to fixing this is to report it, so again, thank you for doing that!

With the goal of keeping the number of tests to a mimum, which form controls do you think should be tested to result in a representative sample? From what you said, it looks like maybe `<select>`?

**Comment by mfairchild365** [comment]
@JamesCatt I added another test for `<select>` and closed this issue. Please comment if you think more testing is needed.
